android/src/com/unciv/app/MultiplayerTurnCheckWorker.kt

```diff
@@ -216,7 +216,7 @@ class MultiplayerTurnCheckWorker(appContext: Context, workerParams: WorkerParame
                         Pair(USER_ID, settings.userId), Pair(CONFIGURED_DELAY, settings.turnCheckerDelay.seconds),
                         Pair(PERSISTENT_NOTIFICATION_ENABLED, settings.turnCheckerPersistentNotificationEnabled),
                         Pair(FILE_STORAGE, settings.server),
-                        Pair(AUTH_HEADER, "Basic ${Gzip.zip(settings.userId)}:${Gzip.zip(settings.passwords[settings.server] ?: "")}"))
+                        Pair(AUTH_HEADER, "Basic ${Gzip.zip("${settings.userId}:${settings.passwords[settings.server] ?: ""}")}"))
```

Comment:
This is just confusing to read, please separate the different parts out into different lines.
val password = settings.passwords[settings.server] ?: ""
val prezippedAuthValue = "${settings.userId}:$password"
val zippedAuthValue = Gzip.zip(prezippedAuthValue)
val auth_header = "Basic $zippedAuthValue"

Reply:
Also, please use one unified function - currently implemented in 3 places